x/bank/internal/keeper/keeper_test.go

```diff
@@ -146,6 +146,9 @@ func TestSendKeeper(t *testing.T) {
 	// negative values.
 	err = sendKeeper.SendCoins(ctx, addr, addr2, sdk.Coins{sdk.Coin{"FOOCOIN", sdk.NewInt(-5)}})
 	require.Error(t, err)
+	events := ctx.EventManager().Events()
```

Comment:
Maybe we should instead create a separate/concise unit test? This test would check the length and the exact events returned?

Reply:
Good idea, I just added it